src/storhaug

```diff
@@ -368,6 +394,49 @@ setup_create_resources()
     pcs -f ${cibfile} constraint location ganesha_state-clone rule resource-discovery=exclusive score=0 role eq storage
 
     pcs cluster cib-push ${cibfile} || storlog "ERR" "Failed to create filesystem resources."
+}
+
+# nothing needed for now
+setup_create_resources_shared_rados_kv()
+{
+    :
+}
+
+setup_create_resources_shared()
```

Comment:
I think this can just be folded into `setup_create_resources_nfs()`. There shouldn't be anything in the future that would require anything for SMB here.